spec/api-crash-reporter-spec.js

```diff
@@ -256,23 +257,31 @@ describe('crashReporter module', () => {
     })
   })
 
+  // TODO(alexeykuzmin): This suite should explicitly
+  // generate several crash reports instead of hoping
+  // that there will be enough of them already.
   describe('getLastCrashReport', () => {
     it('correctly returns the most recent report', () => {
       const reports = crashReporter.getUploadedReports()
+      expect(reports).to.be.an('array')
+      expect(reports).to.have.lengthOf.at.least(2,
+          'There are not enough reports for this test')
+
       const lastReport = crashReporter.getLastCrashReport()
+      expect(lastReport).to.be.an('object').that.includes.a.key('date')
 
-      // Let's find the newest report
-      const newestReport = reports.reduce((acc, cur) => {
+      // Let's find the newest report.
+      const {report: newestReport} = reports.reduce((acc, cur) => {
         const timestamp = new Date(cur.date).getTime()
         return (timestamp > acc.timestamp)
           ? { report: cur, timestamp: timestamp }
           : acc
-      }, { timestamp: 0 })
+      }, { timestamp: -Infinity })
```

Comment:
> `const timestamp = new Date(cur.date).getTime()`

`getTime()` returns an amount of seconds since the Epoch, and can be negative.
The test should work even if a test machine lives in the 60s.